### DISCUSSION TITLE
Kiali UI faced timeout issue intermittently

### DISCUSSION THREAD

**Discussion by toming90** [kiali/kiali#6979] [Q&A] [answered]
Hi,

I have setup Kiali via Kiali operator in our clusters. However we faced following errors in kiali ui pod intermittently, graph and applications cannot list properly:
```2023-12-19T15:15:10Z ERR Error fetching the proxy-status in the following pods: istiod-755f88677f-gx966: context deadline exceeded (Client.Timeout or context cancellation while reading body); istiod-755f88677f-tctc2: context deadline exceeded (Client.Timeout or context cancellation while reading body); istiod-755f88677f-72tvj: context deadline exceeded (Client.Timeout or context cancellation while reading body); istiod-755f88677f-wf62g: context deadline exceeded (Client.Timeout or context cancellation while reading body); istiod-755f88677f-f8gjb: context deadline exceeded (Client.Timeout or context cancellation while reading body); istiod-755f88677f-4w898: context deadline exceeded (Client.Timeout or context cancellation while reading body);```
To me, it seems kiali ui timed out when it tries to fetch data object from istiod. 
We have a qa namespace which includes about 300 deployments. Can kiali handle such number of applications in one namespace? Can we increase the timeout value in kiali side? 

Kiali version: 1.59.0
Istio version: 1.15

**Comment by jshaughn** [comment, marked as answer]
You may want to try updating Kiali to 1.63.2.  I'm not sure if it is compatible with Istio 1.15, but it might be OK.  There was a fix to istiod access.  Also, if the fix is still insufficient there is a new feature to disable the istio api (at the cost of some features).

**Reply by toming90** [reply]
Thanks @jshaughn . From https://kiali.io/docs/installation/installation-guide/prerequisites/ , Kiali 1.63.2 is compatible with Istio 1.16. But I will give it a try and update it here.

**Reply by toming90** [reply]
@jshaughn This issue resolved after I update Kiali to 1.63.2 and disable istio api. Thank you so much

**Reply by aljesusg** [reply]
Hey @toming90 did you update kiali or istio?

**Reply by toming90** [reply]
@aljesusg I upgraded kiali to 1.63.2 only. Istio is still 1.15

**Reply by aljesusg** [reply]
Ok thanks !
